easybuild/easyconfigs/b/Bader/Bader-1.02-intel-2018a.eb

```diff
@@ -0,0 +1,31 @@
+easyblock = 'MakeCp'
+
+name = 'Bader'
+version = '1.02'
+
+homepage = 'http://theory.cm.utexas.edu/henkelman/code/bader/'
+description = "A fast algorithm for doing Bader's analysis on a charge density grid."
+
+toolchain = {'name': 'intel', 'version': '2018a'}
+
+source_urls = ['http://theory.cm.utexas.edu/henkelman/code/bader/download/v%(version)s']
+sources = [{
+    'filename': 'bader-%(version)s.tar.gz',
```

Comment:
@migueldiascosta This is just `SOURCELOWER_TAR_GZ` (and you might as well use `SOURCE_TAR_GZ`)? Also, make this a single line (since it's one source):

```python
sources = [{'download_filename': 'bader.tar.gz', 'filename': SOURCE_TAR_GZ}]